src/content/docs/apm/agents/c-sdk/index.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: C SDK
+type: landingPage
+tags:
+  - Agents
+  - C SDK
+redirects:
+  - /docs/agents/c-sdk/index
```

Comment:
this needs to be `/docs/agents/c-sdk` so just deleting the `index` on the end should fix the 404s. this needs to happen in each of the agent index.mdx files

Reply:
@LizBaker, just as you said, when I removed these "index" references, I was able to build and see the new changes!